gravitee-rest-api-portal/gravitee-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/UserResource.java

```diff
@@ -97,10 +97,23 @@ public Response updateCurrentUser(@Valid @NotNull(message = "Input must not be n
         if(!getAuthenticatedUser().equals(user.getId())) {
             throw new UnauthorizedAccessException();
         }
-        userService.findById(getAuthenticatedUser());
+        UserEntity existingUser = userService.findById(getAuthenticatedUser());
 
         UpdateUserEntity updateUserEntity = new UpdateUserEntity();
-        updateUserEntity.setPicture(checkAndScaleImage(user.getAvatar()));
+        if (user.getAvatar() != null && !user.getAvatar().startsWith("http")) {
```

Comment:
Management REST API expect the format "data:image/...", http comes from the Portail UI when the picture doesn't have changed. I did a change to the portal to ignore the picture if it is an URL but I prefer a double check to avoid bad suprises.
I will add a comment.